src/actors/symbolication.rs

```diff
@@ -1372,7 +1373,8 @@ impl SymbolicationActor {
                             .collect::<CodeModulesBuilder>();
 
                         let minidump_state = MinidumpState {
-                            timestamp: process_state.timestamp(),
+                            timestamp: Utc
+                                .timestamp(process_state.timestamp().try_into().unwrap(), 0),
```

Comment:
is this really never gonna fail?

Reply:
well, if we get a minidump that was created at 2**63 seconds since the epoch it'd fail.  Or if a clock was that far off.  The former won't be for our lifetime (something like 292 trillion years in the future if i got this right?).  The bad clock thing is more interesting I guess, I chose panic based on that this is too far in the future to care, not on the bad clock.  And I reckoned panic would be able to be debugged if it does happen.

However this happens in the procspawned part of the code... so maybe we wouldn't notice this :(